kyaml/kio/byteio_reader.go

```diff
@@ -208,6 +223,10 @@ func (r *ByteReader) Read() ([]*yaml.RNode, error) {
 		}
 
 		if err != nil {
+			if r.WrapBareSeqNode {
```

Comment:
Why ? not clear to me.

Reply:
I think this didn't break any tests in kustomize related to invalid YAML in multi-yaml doc because `WrapBareSeqNode` is not being used in kustomize anywhere, but this will surely break something in kpt.

Reply:
Yes. This check should be removed. Thanks for catching it.